system/core/base.php

```diff
@@ -321,15 +285,17 @@ function hypha_getHtml() {
 		$contents - html file contents
 	*/
 	function hypha_setHtml($contents) {
-		Hypha::$data->html->writeWithLock($contents);
+		global $O_O;
+		$O_O->data->html->writeWithLock($contents);
```

Comment:
This should be `themeHtml` in stead of `html`; `$O_O->data->themeHtml->writeWithLock($contents);`

Reply:
Fixed.